src/api/getFeedbacks.js

```diff
@@ -13,3 +13,4 @@ const getFeedbacks = async () => {
   }
 };
 export default getFeedbacks;
+//spróbować tym sposobem: https://frontendmasters.com/courses/serverless-functions/introduction/
```

Comment:
👀

Reply:
Sorry, it should not be there. It is my idea how to resolve the problem with deploying my page on Netlify - I am working on local JSON data - if I understand well, to deploy my page on Netlify I must use a serverless function (I worked with json-server).